sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/_tests_/e2e/student-scholastic-standings.institutions.controller.saveScholasticStanding.e2e-spec.ts

```diff
@@ -130,12 +134,16 @@ describe("StudentScholasticStandingsInstitutionsController(e2e)-saveScholasticSt
       });
   });
 
-  it("Should throw unprocessable entity exception error when the application change is not eligible.", async () => {
+  it("Should throw unprocessable entity exception error when the application change is not eligible / Archived.", async () => {
```

Comment:
`Should throw unprocessible entity exception error when when the application is archived` ?